libopenage/main.cpp

```diff
@@ -34,6 +34,8 @@ int run_game(const main_arguments &args) {
 	auto &cvar_manager = engine.get_cvar_manager();
 	cvar_manager.load_main_config();
 
+	util::load_csv_files(data_dir.join("converted/meta.docx"));
+
```

Comment:
This is just a side-effect method without returning anything, so we should avoid it. Also, this should not happen in the `main.cpp`, instead in `game_spec.cpp` so it is accounted to the right timer and is also located in the right file.